src/Compression/CompressionCodecDeflateQpl.h

```diff
@@ -108,8 +114,8 @@ class CompressionCodecDeflateQpl : public ICompressionCodec
 private:
     UInt32 getMaxCompressedDataSize(UInt32 uncompressed_size) const override;
 
-    std::unique_ptr<HardwareCodecDeflateQpl> hw_codec;
     std::unique_ptr<SoftwareCodecDeflateQpl> sw_codec;
+    std::unique_ptr<HardwareCodecDeflateQpl> hw_codec;
```

Comment:
Why?

Reply:
The point of switching the order of members in the class was that we could continue to initialize them in the initialization list
[PR in master](https://github.com/ClickHouse/ClickHouse/pull/57291#discussion_r1408928477)

`CompressionCodecDeflateQpl::CompressionCodecDeflateQpl()
    : sw_codec(std::make_unique<SoftwareCodecDeflateQpl>())
    , hw_codec(std::make_unique<HardwareCodecDeflateQpl>(sw_codec))
{
    ...`